Makefile

```diff
@@ -189,29 +189,32 @@ install:
 logcat:
 	$(ADB) logcat | grep -i -E "python|kolibr| `$(ADB) shell ps | grep ' org.endlessos.Key$$' | tr -s [:space:] ' ' | cut -d' ' -f2` " | grep -E -v "WifiTrafficPoller|localhost:5000|NetworkManagementSocketTagger|No jobs to start"
 
-$(SDK)/cmdline-tools:
-	@echo "Downloading Android SDK build tools"
+$(SDK)/cmdline-tools/latest/bin/sdkmanager:
+	@echo "Downloading Android SDK command line tools"
 	wget https://dl.google.com/android/repository/commandlinetools-$(PLATFORM)-7583922_latest.zip
-	unzip commandlinetools-$(PLATFORM)-7583922_latest.zip -d $(SDK)
+	rm -rf cmdline-tools
+	unzip commandlinetools-$(PLATFORM)-7583922_latest.zip
+# This is unfortunate since it will download the command line tools
+# again, but after this it will be properly installed and updatable.
+	yes y | ./cmdline-tools/bin/sdkmanager "cmdline-tools;latest" --sdk_root=$(SDK)
+	rm -rf cmdline-tools
```

Comment:
ok! After going to the Android developers website I understand why is this. They say "You can use the included [sdkmanager](https://developer.android.com/studio/command-line/sdkmanager) to download other SDK packages." So you are using a certain version to download the latest.

Reply:
Right, this is bootstrapping a proper installation. If you look at `$SDK/cmdline-tools/latest` compared to the files in the downloaded zip, you'll see that there are `package.xml` and `source.properties` files. These are the package metadata files generated by `sdkmanager`. Without them, `sdkmanager` doesn't recognize the installed files as a package.

So, you could just unzip the downloaded commandlinetools to the "correct" place in `$SDK/cmdline-tools/latest`, but without the package metadata it causes subtle warnings and errors. For instance, it doesn't figure out the SDK root and instead requires passing `--sdk_root` every time. Similarly, if you ever try to update cmdline-tools, it will not understand the manually installed cmdline-tools and install the new version next to it instead of replacing it.

I wish there was a better way to do this without downloading it twice, but this is the best I could come up with.